core/native/src/Instant.kt

```diff
@@ -234,22 +234,31 @@ public actual class Instant internal constructor(public actual val epochSeconds:
 
 }
 
-private fun Instant.toZonedLocalDateTimeFailing(zone: TimeZone): ZonedDateTime = try {
-    toZonedLocalDateTime(zone)
+private fun Instant.toZonedDateTimeFailing(zone: TimeZone): ZonedDateTime = try {
+    toZonedDateTime(zone)
 } catch (e: IllegalArgumentException) {
     throw DateTimeArithmeticException("Can not convert instant $this to LocalDateTime to perform computations", e)
 }
 
+// org.threeten.bp.LocalDateTime#ofEpochSecond + org.threeten.bp.ZonedDateTime#create
```

Comment:
This comment now makes little sense (`LocalDateTime#ofEpochSeconds` is now used in `Instant.toLocalDateTimeImpl`, not here). Also, maybe inline this function?